### PR TITLE
docs(config): clarify `stats.md` sort fields

### PR DESCRIPTION
**Pull request by Raiondesu**
Added possible values for `assetsSort`, `chunksSort` and `moduleSort` according to the [sources](https://github.com/webpack/webpack/blob/cec5bfb01c025a28062646e18467293e4d6b93ab/lib/Stats.js#L489).

Before this change it was impossible to straight up guess all the possible sorting fields for statistics.

I see this as a critical drawback for projects with a large amount of chunks and assets (I work on one with ~100 chunks). Lack of relevant field sorting documentation means hours wasted on trying to align the records in console statistics.
Like it was for me.

After 2 hours of googling and rereading webpack documentation I had to dive into the source-code.

This PR is here to make webpack docs another tiny bit better and to ensure that nobody will have to forcefully scan the sources in order to do plain-simple things like proper sorting in console.

### CODE REVIEW

**Comment by Raiondesu** [comment]
Well... any comments?

File: src/content/configuration/stats.md [hunk 1]
```diff
@@ -171,3 +178,27 @@ module.exports = {
   }
 }
 ```
+
+### Sorting fields
+
+For `assetsSort`, `chunksSort` and `moduleSort` there are several possible fields that you can sort items by:
+
+- `id` is the item's id;
```
[the remark this answers is not in the document]
Comment (Raiondesu) [reply src/content/configuration/stats.md]: Got it

**Comment by Raiondesu** [comment]
@EugeneHlushko ?

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**